### ISSUE TITLE
BUG:  2_apply_layout.sh overrides wallpaper

### ISSUE THREAD

**Issue by Svenum** [nix-community/plasma-manager#116]
The script 1_apply_themes.sh works perfectly, but if the second script 2_apply_layout.sh runs it changes the wallpaper to the default.

My config:

```nix
workspace = {                                                                                                                                                                                                                                            
      clickItemTo = "select";                                                                                                                                                                                                                                
      theme = "default";                                                                                                                                                                                                                                     
      colorScheme = "CatppuccinMochaTeal";                                                                                                                                                                                               
      cursorTheme = "Catppuccin-Latte-Teal-Cursors";                                                                                                                                                                                    
      wallpaper = /etc/wallpaper/catppuccin-mocha.jpg;                                                                                                                                                               
    };
```

**Comment by magnouvean** [comment]
Quite sure this happens due to the screen-specification of the panels needing to restart kwin. We can probably fix it by setting the wallpaper after creating the panels though. I'll see if that helps anyway.

**Comment by magnouvean** [comment]
It worked. Should be fixed by #118. If the issue persists feel free to reopen.
